src/dev/sdram.cpp

```diff
@@ -304,54 +333,47 @@ static void HAL_FMC_MspInit(void)
+    GPIO_InitStruct.Mode      = GPIO_MODE_AF_PP;
+    GPIO_InitStruct.Pull      = GPIO_NOPULL;
+    GPIO_InitStruct.Speed     = GPIO_SPEED_FREQ_VERY_HIGH;
+    GPIO_InitStruct.Alternate = GPIO_AF12_FMC; // They all seem to use this
+    HAL_GPIO_Init(port, &GPIO_InitStruct);
 
     // This pin can change between boards (SDNWE)
-    //	switch(dsy_sdram.board)
-    //	{
-    //	case DSY_SYS_BOARD_DAISY:
-    //		/* GPIO_InitStruct */
-    //		GPIO_InitStruct.Pin = GPIO_PIN_0;
-    //		GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
-    //		GPIO_InitStruct.Pull = GPIO_NOPULL;
-    //		GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_VERY_HIGH;
-    //		GPIO_InitStruct.Alternate = GPIO_AF12_FMC;
-    //		HAL_GPIO_Init(GPIOC, &GPIO_InitStruct);
-    //		break;
-    //	case DSY_SYS_BOARD_DAISY_SEED:
-    //		/* GPIO_InitStruct */
-    //		GPIO_InitStruct.Pin = GPIO_PIN_5;
-    //		GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
-    //		GPIO_InitStruct.Pull = GPIO_NOPULL;
-    //		GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_VERY_HIGH;
-    //		GPIO_InitStruct.Alternate = GPIO_AF12_FMC;
-    //		HAL_GPIO_Init(GPIOH, &GPIO_InitStruct);
-    //		break;
-    //	case DSY_SYS_BOARD_AUDIO_BB:
-    //		/* GPIO_InitStruct */
-    //		GPIO_InitStruct.Pin = GPIO_PIN_5;
-    //		GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
-    //		GPIO_InitStruct.Pull = GPIO_NOPULL;
-    //		GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_VERY_HIGH;
-    //		GPIO_InitStruct.Alternate = GPIO_AF12_FMC;
-    //		HAL_GPIO_Init(GPIOH, &GPIO_InitStruct);
-    //		break;
-    //	default:
-    //		break;
-    //	}
+    //    switch(dsy_sdram.board)
+    //    {
+    //    case DSY_SYS_BOARD_DAISY:
+    //        /* GPIO_InitStruct */
+    //        GPIO_InitStruct.Pin = GPIO_PIN_0;
+    //        GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
+    //        GPIO_InitStruct.Pull = GPIO_NOPULL;
+    //        GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_VERY_HIGH;
+    //        GPIO_InitStruct.Alternate = GPIO_AF12_FMC;
+    //        HAL_GPIO_Init(GPIOC, &GPIO_InitStruct);
+    //        break;
+    //    case DSY_SYS_BOARD_DAISY_SEED:
+    //        /* GPIO_InitStruct */
+    //        GPIO_InitStruct.Pin = GPIO_PIN_5;
+    //        GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
+    //        GPIO_InitStruct.Pull = GPIO_NOPULL;
+    //        GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_VERY_HIGH;
+    //        GPIO_InitStruct.Alternate = GPIO_AF12_FMC;
+    //        HAL_GPIO_Init(GPIOH, &GPIO_InitStruct);
+    //        break;
+    //    case DSY_SYS_BOARD_AUDIO_BB:
+    //        /* GPIO_InitStruct */
+    //        GPIO_InitStruct.Pin = GPIO_PIN_5;
+    //        GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
+    //        GPIO_InitStruct.Pull = GPIO_NOPULL;
+    //        GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_VERY_HIGH;
+    //        GPIO_InitStruct.Alternate = GPIO_AF12_FMC;
+    //        HAL_GPIO_Init(GPIOH, &GPIO_InitStruct);
+    //        break;
+    //    default:
+    //        break;
+    //    }
```

Comment:
We can get rid of this at this point -- we don't handle different boards this way anymore, and aside from some abandoned/out of date hardware all boards use the same RAM pins now.